staging/src/k8s.io/code-generator/cmd/applyconfiguration-gen/generators/applyconfiguration.go

```diff
@@ -139,6 +139,25 @@ func blocklisted(t *types.Type, member types.Member) bool {
 	return false
 }
 
+func isAtomicList(m types.Member) bool {
+	if m.Type.Kind != types.Slice {
+		return false
+	}
+	if m.Type.Elem == types.Byte {
+		return true
+	}
+	listType := types.ExtractCommentTags("+", m.CommentLines)["listType"]
+	return listType != nil && listType[0] == "atomic"
```

Comment:
So something I forgot to mention is that atomic is actually the default for list, so this condition is going to miss a few. Notably that should get all of the `types.Byte` atomic more-or-less automatically making the condition useless.